.github/workflows/ngsa.yaml

```diff
@@ -3,7 +3,7 @@ name: Build NGSA-App
 on:
   schedule:
     # Run daily at 6:00 AM UTC
-    - cron: "0 6 * * *"
+    - cron: "0 6 * * 0"
```

Comment:
quick question, If I am reading this correctly , now want the job to run only once per week on Sunday instead every day?

Reply:
Update the comment if schedule is to change to Sundays

Reply:
resolved